rollup.config.ts

```diff
@@ -37,39 +37,39 @@ export default function rollup(options: RollupOptions): RollupOptions[] {
       name: 'query-core',
       packageDir: 'packages/query-core',
       jsName: 'QueryCore',
-      outputFile: 'query-core',
+      outputFile: 'index',
       entryFile: 'src/index.ts',
       globals: {},
     }),
     ...buildConfigs({
       name: 'query-async-storage-persister',
       packageDir: 'packages/query-async-storage-persister',
       jsName: 'QueryAsyncStoragePersister',
-      outputFile: 'query-async-storage-persister',
+      outputFile: 'index',
```

Comment:
was `outputFile` not used at all previously?

Reply:
It was not used previously, from what I could tell